src/pages/javascript/switch-statements/index.md

```diff
@@ -2,14 +2,71 @@
 title: Switch Statements
 ---
 ## Switch Statements
+<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+A `switch` statement in programming is similar to an `if-else` statement, but has the benefit of sometimes being easier to read when there are a lot of conditions, and it allows a `default` block, that will be executed if none of the other conditions are true.
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/javascript/switch-statements/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+#### Syntax:
+```
```

Comment:
Let's add syntax highlighting to the code examples (this comment applies to the code fencing down below, too). You can do this by changing this opening triple-backtick to: ` ```javascript `

Reply:
Fixed, thanks!